agent/src/softwarecontaineragent.cpp

```diff
@@ -48,7 +48,6 @@ bool SoftwareContainerAgent::triggerPreload()
 inline bool SoftwareContainerAgent::isIdValid (ContainerID containerID)
 {
     return ((containerID < UINT32_MAX)
-            && (containerID >= 0)
```

Comment:
Warning will be there for sure but i expect it to be compiled anyway. In any case you are right, the line should be removed.